src/benchmarklib/benchmark_utils.hpp

```diff
@@ -25,7 +25,12 @@ using TimePoint = std::chrono::high_resolution_clock::time_point;
 using NamedQuery = std::pair<std::string, std::string>;
 using NamedQueries = std::vector<NamedQuery>;
 
+<<<<<<< HEAD
```

Comment:
still broken

Reply:
jup, I accidentally open the PR while I wasn't ready yet :/ give me a sec

Reply:
@mrks should be ready now